### ISSUE TITLE
Properites & Arguments marked as `support.function`

### ISSUE THREAD

**Issue by evoactivity** [lifeart/vsc-ember-syntax#46]
| Enabled | Disabled |
|--------|-----|
|![image](https://github.com/lifeart/vsc-ember-syntax/assets/83799/be7da7ed-dbd7-465b-9f05-839f3351964b)|![image](https://github.com/lifeart/vsc-ember-syntax/assets/83799/93942ec3-0160-40fe-b054-3728a856a527)|

I like that data access is colored but I don't like that it is marked as a function, had reports this is putting some people off using this extension.

Not sure when I'll be able to dig into this but just creating an issue so I don't forget.

**Comment by lifeart** [comment]
Hi @evoactivity, thank you for report, but it's only about "how it's looks like", I skipping semantic here prior to colors.
In textmate scope is matched to colors, and it's really hard to get proper color and relative scope :)

**Comment by evoactivity** [comment]
I think the problem is these are not actually functions, so shouldn't be highlighed as functions at all. You can see with the extension disabled they are properly marked as variables/parameters.

This is where that is happening in the syntax https://github.com/lifeart/vsc-ember-syntax/blob/55ca6981204a1f3934b55057a5a7abbab66a1d8e/syntaxes/handlebars.tmLanguage.json#L210

**Comment by lifeart** [comment]
I don't see problem here, I men, if color is confusing - it could be manually changed.
You may try to specify 'different' scopes, but it likely gonna look so sad (in terms of colouring)

**Comment by evoactivity** [comment]
I agree to be honest, visually I think it's fine as it is. I was told though people are still using the unmaintained `vscode-glimmer` because they don't like the property access being the same colour as a js/ts function when they are not functions. I would like to remove roadblocks to people using this extension, even if it seems trivial. 

You're probably right and I might not be able to find a satisfying approach to colour them differently, but I'd still like to play around with it and see if I can't do something that satisfies everyone.

**Comment by lifeart** [comment]
@evoactivity you may propose coloring ovverides (as in readme) and we could offer it as `themes`.

**Comment by evoactivity** [comment]
Cool. I won't have time to visit this immediately but it's in my backlog :)
